src/documenti/esterni/AdR/contenuti/casi_uso.tex

```diff
@@ -194,11 +195,24 @@ \section{UC6 - Salvataggio Sessione}
   \end{enumerate}
 \end{itemize}
 
-\section{UC9- Gestione errore}
+\section{UC9- Visualizzazione errore inserimento dati}
+\begin{itemize}
+  \item \textbf{Attore Primario:} Utente;
+  \item \textbf{Precondizioni:} L’utente carica un file CSV contenente i dati da analizzare mal formattato o
```

Comment:
Secondo me le precondizioni e le post condizioni dicono poco sullo stato del sistema. Si capisce solo nel momento in cui uno vede che UC9 estende UC1 però non so come possa venire valutata questa cosa.

(Questa è solo una nota per discutere sulla AdR in generale, non è bloccante per questa pull request)